{
    "config": {
        "abort": {
            "already_configured": "Perangkat sudah dikonfigurasi",
            "cannot_connect": "Tidak dapat terhubung ke Volumio yang ditemukan"
        },
        "error": {
            "cannot_connect": "Gagal terhubung",
            "unknown": "Kesalahan yang tidak diharapkan"
        },
        "step": {
            "discovery_confirm": {
                "description": "Ingin menambahkan Volumio (`{name}`) ke Home Assistant?",
                "title": "Volumio yang ditemukan"
            },
            "user": {
                "data": {
                    "host": "Host",
                    "port": "Port"
                }
            }
        }
    }
}